Documentation/ja/notation/staff.itely

```diff
@@ -1540,3 +1543,108 @@ bassoonNotes = \relative c {
 @unnumberedsubsubsec 空の小節を圧縮する
 @translationof Compressing empty measures
 
+@cindex multi-measure notes, expanding (複数小節にまたがる音符を展開する)
+@cindex multi-measure notes, contracting (複数小節にまたがる音符をまとめる)
+@funindex \compressEmptyMeasures
+@funindex \expandEmptyMeasures
+
+デフォルトでは、すべての小節は空であっても譜刻されます --
+これは、リズムイベント (音符、休符、非表示の休符など)
+が複数の小節にまたがるほど長い場合に発生する可能性があります。
+この動作は、次の図に示すように、@c
+すべての空の小節を単一の小節にまとめることで変更できます
+(この例の 2 番目の部分は小節を展開して実際にはデフォルトの動作に戻ります):
+
+@lilypond[quote,verbatim,fragment]
+\override Score.BarNumber.break-visibility = #'#(#f #t #t)
+\time 2/4
+\compressEmptyMeasures
+c'1 d'1. e'\breve
+\bar "||"
+\expandEmptyMeasures
+c'1 d'1. e'\breve
+@end lilypond
+
+@noindent
+その表記法は構文的には正しいですが、前の例で示したように、@c
+音楽的な観点からは混乱するかもしれません。@c
+したがって、@ref{Using break-visibility} で説明されている構文を使用して、@c
+小節番号を明示的に譜刻する必要があります。
+
+@cindex multi-measure rests, expanding (複数小節にまたがる休符を展開する)
+@cindex multi-measure rests, contracting (複数小節にまたがる休符をまとめる)
+@funindex \compressMMRests
+
+そのような表記がより有用になる可能性があるのは、@ref{Full measure rests} に適用される場合です。@c
+複数小節にまたがる休符は、複数小節にまたがる休符の記号を含む単一の小節として表示され、@c
+休符の小節数が小節の上に譜刻されます。
+
+@c KEEP LY
+@lilypond[quote,verbatim,fragment]
+% デフォルトの振る舞い
+\time 3/4 r2. | R2.*2 |
+\time 2/4 R2 |
+\time 4/4
+% 休みの小節を 1 つの小節にまとめます
+\compressEmptyMeasures
+r1 | R1*17 | R1*4 |
+\expandEmptyMeasures
+% 休みの小節を展開します
+\time 3/4
+R2.*2 |
+@end lilypond
+
+@code{\compressEmptyMeasures} とは異なり、@c
+音楽関数 @code{\compressMMRests} は休符にのみ適用され、@c
+他のイベントは圧縮されません。@c
+プロパティ設定ではなく関数として、@c
+その構文は音楽表現が続く必要があるという点でわずかに異なります:
+
+@lilypond[quote,verbatim,fragment]
```

Comment:
セルフコメントで失礼します。
`@c KEEP LY` という記述って、ここにも必要でしょうか？

Reply:
KEEP LY付きでお願いします。

http://lilypond.org/doc/v2.19/Documentation/contributor/documentation-translation-details

> Please keep verbatim copies of music snippets (in @lilypond blocs). However, some music snippets containing text that shows in the rendered music, and sometimes translating this text really helps the user to understand the documentation; in this case, and only in this case, you may as an exception translate text in the music snippet, and then you must add a line immediately before the @lilypond block, starting with
`@c KEEP LY`
Otherwise the music snippet would be reset to the same content as the English version at next make snippet-update run

lilypondコード中のコメント等を日本語に翻訳する場合に必要になります。